pg_search/Cargo.toml

```diff
@@ -14,6 +14,7 @@ pg13 = ["pgrx/pg13", "pgrx-tests/pg13"]
 pg14 = ["pgrx/pg14", "pgrx-tests/pg14"]
 pg15 = ["pgrx/pg15", "pgrx-tests/pg15"]
 pg16 = ["pgrx/pg16", "pgrx-tests/pg16"]
+pg17 = ["pgrx/pg17", "pgrx-tests/pg17"]
```

Comment:
I wonder if we should default to PG17 right away or not... Perhaps in a follow on PR

Reply:
If you want to do it in this PR, we need to do `pg17` on line 12

Reply:
Let's wait a few weeks after rolling out PG17 builds. By changing this and lint workflow files it should be good to go